client/src/pages/Comparison.tsx

```diff
@@ -71,10 +71,31 @@ export function Comparison() {
     return <div style={{ color: 'white' }}>Loading...</div>;
   }
 
-  if (error || !mostCompatibleAll) {
+  if (error || !mostCompatibleAll || !mostCompatibleAll.highestUserId) {
     return (
-      <div style={{ color: 'white' }}>
-        Rate more movies to see your accurate compatibility!
+      <div className="comparison-wrapper">
+        <h1>COMPARISON</h1>
+        <div
+          className="row"
+          style={{ justifyContent: 'start', alignItems: 'flex-end' }}>
+          <h2
+            className={'selected'}
+            style={{ padding: '1rem' }}
+            onClick={() => setIsAllView(true)}>
+            All Users
+          </h2>
+          <h2
+            className={'selected'}
+            style={{ padding: '1rem' }}
+            onClick={() => setIsAllView(false)}>
+            Following
+          </h2>
+        </div>
+        <hr />
+        <h3 style={{ color: 'white' }}>
+          {' '}
```

Comment:
Are these explicit spaces necessary? Often they are put here by prettier during a refactoring. They don't always make a difference to the display. If not, it's best to remove them.